.genx/templates/main/main.hbs

```diff
@@ -1,3 +1,6 @@
+{{#if data.fdc3ListenChannel}}
+import { listenToChannel } from '../utils';
+{{/if}}
 import { Connect, ConnectConfig, defaultConnectConfig } from '@genesislcap/foundation-comms';
```

Comment:
This rename/move shouldn't be needed - I can sort out in my branch though

Reply:
It's not just `fdc3` it's a more specific action

Reply:
I meant moving of `client/src/main/main.ts` to `.genx/templates/main/main.hbs`.